jbt-it/neomdb
# 246 server implement automatic database setup

- Added a script for filling the database with dummy data.
- Upgraded to mysql2 -> This led to a different evaluation of types, this is why QueryResult is used instead of custom types. 
 
**How to test:**
- Start mysql-docker
- Run `npm run filldb` to start the database script
- Check if dummy data is inserted in the database (look at some of the insert statements in server/src/db-scripts/fill_db_commands.sql and compare it to the database)
